addon/components/freestyle/bootstrap/form/-controls.js

```diff
@@ -1,14 +1,6 @@
 import Component from '@ember/component';
 import layout from '../../../templates/components/freestyle/bootstrap/form/-controls';
-import { BuilderForPropDefaults } from 'ember-bootstrap-controls/utils/prop-definition-tools';
-import { propDefinitions } from '../../bootstrap/form/-controls';
 
 export default Component.extend({
   layout,
```

Comment:
Should this have an implicit wrapping `div`? If not desired, I recommend a `tagName: ''` here as well.